adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/jaas/AbstractKeycloakLoginModule.java

```diff
@@ -84,7 +85,15 @@ public void initialize(Subject subject, CallbackHandler callbackHandler, Map<Str
 
     protected KeycloakDeployment resolveDeployment(String keycloakConfigFile) {
         try {
-            InputStream is = FindFile.findFile(keycloakConfigFile);
+        	    InputStream is = null;
+				// Added support for loading resource from Profile
+				if (keycloakConfigFile.contains("profile:")) {
+					is = new URL(keycloakConfigFile).openStream();
+				}
+				else {
+					is = FindFile.findFile(keycloakConfigFile);
+				}        	
+            
```

Comment:
No, I don't think it's a good idea to explicitly hard code "profile:" here.

The problem is simply that findFile should only load from local file system, and there are places in code that already use it and expect such behavior. Expanding existing behaviour to potentially load from network could potentially represent a security risk. But if you have a method that obviously can load as a url that's now a different matter. So you could simply implement resolveDeployment as something like:
```
InputStream is = null;
try {
  is = FindFile.findFile(keycloakConfigFile);
} catch (IOException e) {
  is = new URL(keycloakConfigFile).openStream();
}
```
Or, @stianst, maybe we want to make sure that we don't load any http:// https:// ftp:// urls here?

You also should add a unit test for this. So if someone changes this code it doesn't break new behaviour you'll be relying on.

Also, please use four spaces for indentation in line with the rest of Keycloak code.